.travis.yml

```diff
@@ -7,9 +7,9 @@ cache:
   directories:
   - node_modules
 node_js:
+  - '9'
   - '8'
   - '6'
-  - '4'
```

Comment:
This seems unrelated? We are we dropping node 4 testing already?

Reply:
Can use `node` environment and keep 4? (like in babel-upgrade)

Reply:
If it's because of jest, it still works on node 4 if you use the node environment - https://github.com/babel/babel-upgrade works fine with it.

Reply:
ok cool, I revert then.